### ISSUE TITLE
Expect.within for Relative does not work for negative values

### ISSUE THREAD

**Issue by jmedding** [elm-community/elm-test#211]
`99.9 |> Expect.within (Relative 0.01) 100`

 will pass, but

`-99.9 |> Expect.within (Relative 0.01) -100`

will fail

**Comment by drathier** [comment]
I have a lot of other things to do first, so I won't fix this before Friday, and probably not until next weekend. If someone wants to fix it earlier than that, please go ahead. We're obviously missing test-cases for negative relative numbers, so I'd guess this affects Absolute as well. 

I'd write more tests first, and make sure there are tests for all incorrect cases before going about fixing them.

**Comment by jmedding** [comment]
I created a pull request.

**Comment by drathier** [comment]
Fixed in #212
